src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -351,22 +372,20 @@ public override IEnumerable<TResult2> Select<TResult2>(Func<TResult, TResult2> s
             {
```

Comment:
We used to have test coverage for this method.  With these changes, this method is now uncovered by tests.